cms_status_message/models.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
+# © 2017 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openerp import models, api
+from openerp.http import request
+from openerp import _
+
+
+class Website(models.Model):
+    _inherit = 'website'
+
+    @property
```

Comment:
I'm not sure this will work fine... Did you check if strings get correctly translated? I also don't know if this can be easily inherited/extended... 🤔